src/libs/actions/PaymentMethods.js

```diff
@@ -95,8 +123,42 @@ function clearDebitCardFormErrorAndSubmit() {
     });
 }
 
+/**
+ * Set the necessary data for wallet transfer
+ * @param {Number} currentBalance
+ * @param {Number} selectedAccountID
+ */
+function saveWalletTransferAmountAndAccount(currentBalance, selectedAccountID) {
```

Comment:
This method is used for sharing data across multiple screens.

Reply:
Why are the screens not able to share this data?

Reply:
Does this mean we will have something like a "local transfer" that is temporarily stored until we are ready to make an API request? Let's try to be as descriptive as possible here please. I can help write the comment but I'm confused about why we even need this.

Reply:
Actually, screens change based on the Route and thus I am following the pattern of using Onyx to share data among navigation screens.

Reply:
Ok, I think in that case this method name sounds too much like something that would call an API?

What about splitting these up into more specific methods like:

`setWalletTransferAmount()` and `setWalletTransferAccountID()`

With a comment like:

>The Transfer Balance flow requires the use of several different screens. These methods enables us to share data about the current transfer between those screens.

Probably outside the scope of this issue, but kind of feels like we should have some distinction between "actions" that call APIs and "actions" that just set data.

Reply:
> Probably outside the scope of this issue, but kind of feels like we should have some distinction between "actions" that call APIs and "actions" that just set data.

I don't feel the urge to do it. Probably follow a common naming convention for such functions.